infrastructure-playbooks/purge-cluster.yml

```diff
@@ -321,6 +326,22 @@
       - ceph_disk_present.rc == 0
       - ceph_data_partlabels.rc == 0
 
+  - name: remove osd logical volumes
```

Comment:
can you add a comment here so that we know this needs to go away once `ceph-volume lvm zap` becomes available?